.gitignore

```diff
@@ -20,3 +20,4 @@ deploy_key
 secrets.tar
 secrets
 htmlcov
+target
```

Comment:
Minor: as we’re adding more languages to the project, I wonder if it would be useful to start adding comments to the `.gitignore` to explain which language a particular ignore is associated with?